api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1427,7 +1427,10 @@ message NodeLogin {
 }
 
 // AdminAction specifies an admin action.
-message AdminAction {}
+message AdminAction {
+  reserved 1;
+  reserved "Name";
```

Comment:
I'm confused, where is the original declaration for Name?

Reply:
It's right here in the diff?

Reply:
I swear GitHub was showing me an empty message:

```diff
-message AdminAction {}
+message AdminAction {
+    reserved 1;
+    reserved "Name";
+}
```

It looks alright now, so nevermind.